EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs

```diff
@@ -43,40 +38,52 @@ public AdministratorsManagePage(IWebDriver driver) : base(driver)
         [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
         private IWebElement _inputPhoneNumberValidationWarning;
 
+        public void WaitAndSendKeysToInputName(string name, int TimeToWait)
```

Comment:
please rename the parameter to **t**imeToWait, parameter's name begins from lower case